pyboy/plugins/debug.py

```diff
@@ -8,11 +8,13 @@
 from array import array
 
 import sdl2
+from sdl2.sdlttf import *
```

Comment:
Could you expand the `*` to be the explicit imports? It makes it easier to work with in the future.